load_hours.py

```diff
@@ -10,14 +10,26 @@
 harvest_client = HarvestClient()
 jira_client = JiraClient()
 
-date_from = date.today().isoformat()
-date_to = date.today().isoformat()
+# Track the last day it ran
+with open('.aux', 'r') as f:
```

Comment:
I think we can create wrap this `get last day logic to a function` for better readability.